qemu/tests/cfg/timerdevice.cfg

```diff
@@ -12,10 +12,13 @@
             hwclock_time_format = "%a %b %d %H:%M:%S %Y"
     variants:
         - tscwrite:
+            only Linux
             no RHEL.6
             type = timerdevice_tscwrite
-            msr_tools_install_cmd = "yum install -y msr-tools"
-            msr_tools_cmd = "a=$(rdmsr -d 0x00000010); echo -e ibase=16\\n $a  | tail -n 1 | while read n; do wrmsr 0x00000010 $(($n+100000000000)); echo $n; done"
+            tarball = "tscwrite/msr-tools-1.1.2.tar.gz"
+            msr_path = "/tmp/msr-tools-1.1.2"
```

Comment:
The path should not be defined here, cause you hard code `/tmp` in the script. You might either define the directory and complete path together here or there.

Reply:
Updated.